hips/draw/tests/test_simple.py

```diff
@@ -53,5 +55,34 @@ def test_make_sky_image():
     assert_allclose(data[200, 994], 3717.10091363)
     assert_allclose(data[200, 995], 3402.55292158)
 
+get_hips_order_pars = [
+    dict(order=7, width=2000, height=1000, fov="3 deg"),
```

Comment:
To test the functionality here you can simply use one fixed `width` and `height` (e.g. 200 and 100). All you need to vary and list here is some parameter that changes the image pixel size like e.g. `fov`.